src/test/java/seedu/address/model/TutorsPetTest.java

```diff
@@ -42,13 +51,16 @@ public void resetData_withValidReadOnlyTutorsPet_replacesData() {
         assertEquals(newData, tutorsPet);
     }
 
+    //// student-related tests
```

Comment:
Would this be a better name?
```suggestion
    //// Student related tests
```

Reply:
```//// student-related tests``` may work better here instead of Student, since we also have ```//// moduleClass-related tests```

Reply:
Agree with @junlong4321 for consistency.